source/_integrations/imap.markdown

```diff
@@ -220,6 +276,44 @@ actions:
 
 {% endraw %}
 
+In case you want want to process a message part, you need to know the part index and specify it via the `part` option of the `fetch` action. 
```

Comment:
```suggestion
In case you want to process a message part, you need to know the part index and specify it via the `part` option of the `fetch` action. 
```